server/src/main/java/com/cloud/alert/SecondaryStorageVmAlertAdapter.java

```diff
@@ -59,92 +59,100 @@ public void onSSVMAlert(Object sender, SecStorageVmAlertEventArgs args) {
             throw new CloudRuntimeException("Invalid alert arguments, secStorageVm must be set");
         }
 
+        String secStorageVmHostName = "";
+        String secStorageVmPublicIpAddress = "";
+        String secStorageVmPrivateIpAddress = "N/A";
+        Long secStorageVmPodIdToDeployIn = null;
+
+        if (secStorageVm != null) {
+            secStorageVmHostName = secStorageVm.getHostName();
+            secStorageVmPublicIpAddress = secStorageVm.getPublicIpAddress();
+            secStorageVmPrivateIpAddress = secStorageVm.getPrivateIpAddress() == null ? "N/A" : secStorageVm.getPrivateIpAddress();
+            secStorageVmPodIdToDeployIn = secStorageVm.getPodIdToDeployIn();
+        }
```

Comment:
this is the exact same functionality as https://github.com/apache/cloudstack/pull/7140/files#diff-836e2e853d64b7a2f86c1add5aa76deea112788fe69c19a3c5eb2618d753ff2c lines 63-73. Can we make it a utiity method?

i.e. can we use `VMInstanceVO` instead of `ConsoleProxyVO` and `SecondaryStorageVmVO` and make it a member method of that class.

Reply:
Hey @DaanHoogland, yes this is replicate code from the ConsoleProxyAlertAdapter, but is it worth it to make a utility method out of it? 
To make the code clear I would probably have to create a new object to store this info and use it during these methods. In the end I would probably end up with more lines of code and a whole new class just for this.

Reply:
@JoaoJandre , I am just bringing it to your attention. If you feel it isn't worth the trouble and you don't think it will be used more than considder my remark answerred.
Just one more opiniated remark; I think it is worth even if it doesn´t save lines of code to unify it. It makes clear to the reader that the same is happening, as future readers will not see the two peaces at the same time as we do now. This can also be solved by a comment of course.